### ISSUE TITLE
Handle code blocks using uppercase letters

### ISSUE THREAD

**Issue by IonicaBizau** [atom/language-gfm#109]
Just letting you know that ```` ```JS ```` blocks are not highlighted.

![image](https://cloud.githubusercontent.com/assets/2864371/8132648/de452872-112b-11e5-84cf-9007902f8431.png)

On both GitHub & Gist, they are:

```JS
// this uses ```JS
function foo () {
  return 42;
}
```

```js
// and this lowercase: ```js
function foo () {
  return 42;
}
```

> ![image](https://cloud.githubusercontent.com/assets/2864371/8132740/8c7fd07c-112c-11e5-879f-27de488592be.png)

This is indeed because of [this](https://github.com/atom/language-gfm/blob/7cb930dcaf387c9eb2b40e8c20a723c52d717c55/grammars/gfm.cson#L219). Is it possible to make these regular expressions case-insensitive?

If not, a VIM macro can copy them to contain `js|JS` and so. :laughing:

I'd be glad to push some contributions, but before doing so, like always, it's better to discuss.

**Comment by IonicaBizau** [comment]
@izuzak Can you please take a look here? :smile_cat:

**Comment by izuzak** [comment]
I think it would be neat to make the highlighting work for both `JS` and `js`, but wouldn't call it high priority. 

Not sure what the best approach would be. @kevinsawicki would using the [ignore case option](http://www.geocities.jp/kosako3/oniguruma/doc/RE.txt) on those groups be the way to go here?

**Comment by kevinsawicki** [comment]
> would using the ignore case option on those groups be the way to go here?

Yup, should be fine to do that

**Comment by IonicaBizau** [comment]
@izuzak @kevinsawicki  Addressed in #123 :sparkle:
